dali/test/python/test_torch_pipeline_rnnt.py

```diff
@@ -355,77 +354,161 @@ def test_torch_vs_dali_normalize():
         for normalize_type in ['per_feature', 'all_features']:
             yield _testimpl_torch_vs_dali_normalize, normalize_type, device
 
+def torch_frame_splicing(mel_spec, stacking=1, subsampling=1, device='cpu'):
+    mel_spec = torch.tensor(mel_spec, dtype=torch.float32).unsqueeze(0)
+    if device == 'gpu':
+        mel_spec = mel_spec.cuda()
+    out = stack_subsample_frames(mel_spec, stacking=stacking, subsampling=subsampling)
+    out = out.cpu().detach().numpy().squeeze(0)
```

Comment:
Why you need to transfer it to the CPU and use numpy squeeze?
Can't you leave it at the GPU and use torch squeeze?

Reply:
Only for convenience. This functions process a single sample, the input is a numpy array and the output as well. This way I can compare torch and DALI implementations side by side. This function is meant as a utility to test every step of the pipeline separately.

Reply:
I see that detach is not here. I'll remove.
squeeze(0) is there because the torch function work with a batch and I want to process a single sample, so I unsqueeze(0) before feeding the function and then squeeze(0) the result to create the batch dimension.